### PR TITLE
Added installation instruction

### LINKED ISSUE #80: Add installation instructions

### PR DESCRIPTION
**Pull request by Nyholm**
This will fix #80

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -34,13 +34,41 @@ $mailer = new Stampie\Mailer\SendGrid($adapter, 'username:password');
...
+
+## Installation
+
+Stampie is not hard coupled to Guzzle or any other library that sends HTTP messages. It uses an abstraction 
+called HTTPlug. This will give you the flexibility to choose what PSR-7 implementation and HTTP client to use. 
+
+If you just want to get started quickly you should run the following command: 
+
+```bash
+php composer.phar require stampie/stampie php-http/curl-client php-http/message guzzlehttp/psr7
```
Comment (stof) [inline README.md]: I would use `composer require` instead. AFAIK, it is most common to document it using a global composer command instead.

**Comment by stof** [comment]
@Nyholm please fix conflicts

File: README.md [hunk 2]
```diff
@@ -34,13 +34,42 @@ $mailer = new Stampie\Mailer\SendGrid($adapter, 'username:password');
...
+### Why requiring so many packages?
+
+Stampie has a dependency on the virtual package [php-http/client-implementation](https://packagist.org/providers/php-http/client-implementation) 
+which requires to you install **an** adapter, but we do not care which one. That is an implementation detail in your application. 
+We also need **a** PSR-7 implementation and **a** message factory. 
+
+You do not have to use the `php-http/curl-client` if you do not want to. You may use the `php-http/guzzle6-adapter` or any
+other library in [this list](https://packagist.org/providers/php-http/client-implementation). 
+Read more about the virtual packages, why this is a good idea and about the flexibility it brings at the [HTTPlug docs](http://docs.php-http.org/en/latest/httplug/users.html).
+=======
```
Comment (stof) [inline README.md]: this line looks wrong to me

**Comment by Nyholm** [comment]
Thank you for the review

**Comment by Nyholm** [comment]
Thank you

//Tobias Nyholm

Skickat på språng

**Pull request merged**